dask_kubernetes/operator/daskcluster.py

```diff
@@ -99,7 +104,11 @@ def build_worker_pod_spec(name, namespace, image, n, scheduler_name):
                 {
                     "name": "scheduler",
                     "image": image,
-                    "args": ["dask-worker", f"tcp://{scheduler_name}.{namespace}:8786"],
+                    "args": [
+                        "dask-worker",
+                        f"tcp://{scheduler_name}.{namespace}:8786",
+                        f"--name={scheduler_name}-{name}-worker-{n}",
```

Comment:
The name is already being formatted on line 95. It would be best to assign it to a variable and reuse it in both places to avoid one getting updated and not the other in the future.